src/features_x86.h

```diff
@@ -72,13 +80,14 @@ JL_FEATURE_DEF(movdir64b, 32 * 3 + 28, 0)
 JL_FEATURE_DEF(enqcmd, 32 * 3 + 29, 0)
 
 // EAX=7,ECX=0: EDX
-// JL_FEATURE_DEF(avx5124vnniw, 32 * 4 + 2, ?????)
-// JL_FEATURE_DEF(avx5124fmaps, 32 * 4 + 3, ?????)
+JL_FEATURE_DEF(avx512vnniw, 32 * 4 + 2, 0)
+JL_FEATURE_DEF(avx512fmaps, 32 * 4 + 3, 0)
 JL_FEATURE_DEF(uintr, 32 * 4 + 5, 140000)
```

Comment:
Isn't the last statement a comment which LLVM version introduced support?

Reply:
As it turns out those were never implemented :)